.travis.yml

```diff
@@ -33,10 +33,10 @@ install:
   - conda install -c conda-forge iris=1.13
 
   # Force numpy to remain at version 1.13.3
```

Comment:
Maybe just remove these lines?

Reply:
I'm still messing around for now. The sphinx version was changed to 1.7.1 in the conda repository, but seems like a slightly broken implementation (this only changed a few days ago); the -d flag is still in the docs as taking a path for version 1.8, but it keeps rejecting that and demanding an integer representing depth (as if it is receiving -D). We are also now getting a lot of segmentation fault issues with the travis build. Removing the discrete call to numpy fixes this issue. I have moved the version specification into the other line without issue; the default numpy version brought in as an iris dependency is now 1.14.

I'm going to tidy the history of this PR to leave only the required fixes.